flake.nix

```diff
@@ -2,79 +2,107 @@
   inputs = {
     crane = {
       url = "github:ipetkov/crane";
-      inputs.nixpkgs.follows = "nixpkgs";
     };
 
     nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
-    utils.url = "github:numtide/flake-utils";
     nixtract.url = "github:tweag/nixtract";
   };
 
   outputs =
     { self
     , crane
     , nixpkgs
-    , utils
     , nixtract
     }:
-    utils.lib.eachDefaultSystem
-      (system:
-      let
-        pkgs = import nixpkgs { inherit system; };
-        crane-lib = crane.mkLib nixpkgs.legacyPackages.${system};
-        cyclonedx = pkgs.cyclonedx-cli;
-        nixtract-cli = nixtract.defaultPackage.${system};
 
+    let
+      # Matches pkgs.tree-sitter
```

Comment:
Any particular reason we care about tree-sitter?

Reply:
Ah, you caught me red-handed copying stuff someone did to get rid of flake-utils from Topiary :upside_down_face: but no, nothing to do with that (and hi :wave: hope you're doing well !)